qa/update-tests/pom.xml

```diff
@@ -22,6 +22,13 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>com.github.docker-java</groupId>
+      <artifactId>docker-java-api</artifactId>
+      <version>3.3.2</version>
```

Comment:
:x: You can remove it, it will be inherited from the parent POM

Reply:
I got a Java check validation error when it was missing, as we make direct use of the classes but lacked the dependency.

Reply:
would you prefer adding the docker-java lib to `ignoredUsedUndeclaredDependencies` instead? 🤔  this way we don’t lock the version, I tend to prefer this then

Reply:
totally missed the parent part 🤦  don't get why it fails then either, will investigate, this was the failing run
https://github.com/camunda/zeebe/actions/runs/5557398956/jobs/10151137919

Reply:
it's only declared in `dependencyManagement` of the parent, will thus still add it but omit the version

Reply:
Sorry, I meant specifically removing the version line :sweat_smile: